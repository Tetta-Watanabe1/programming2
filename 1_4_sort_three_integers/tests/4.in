37 36 82
