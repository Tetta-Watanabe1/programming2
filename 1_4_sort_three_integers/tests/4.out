36
37
82
